sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -176,23 +187,39 @@ def tune(
         packages_to_install: List[str] = None,
```

Comment:
To be implemented: Update `resources_per_trial` to enable PyTorch distributed training.